numpy-stubs/__init__.pyi

```diff
@@ -420,8 +421,21 @@ class int64(signedinteger): ...
 
 class timedelta64(signedinteger):
     def __init__(self, _data: Any = ..., _format: str = ...) -> None: ...
-    def __add__(self, other: timedelta64) -> timedelta64: ...
-    def __sub__(self, other: timedelta64) -> timedelta64: ...
+    @overload
+    def __add__(self, other: Union[timedelta64, int]) -> timedelta64: ...
+    @overload
+    def __add__(self, other: datetime64) -> datetime64: ...
+    def __sub__(self, other: Union[timedelta64, int]) -> timedelta64: ...
+    if sys.version_info[0] < 3:
```

Comment:
```suggestion
    if sys.version_info.major < 3:
```

Reply:
`flake8` doesn't like those - it seems like mypy only supports (or at least flake8 thinks that mypy only supports) changing types when switching on the system version in the old style...

Reply:
;(